docs/cs-manual/source/record-filters.rst

```diff
@@ -0,0 +1,277 @@
+   effect on the game itself.
+
+Filter
+   The actual contents of the filter are given here using the filter syntax.
+   Change the expressions to modify what the filter returns.
+
+Description
+   A textual description of what the filter does.
+
+
+
+Using predefined filters
+************************
+
+To use a filter you have to type its ID into the filter field above a table.
+
+For instance, try to opening the objects table (under the world menu) and type
+into the filters field ``project::weapons``. As soon as you complete the text
+the table will show only the weapons. The text ``project::weapons`` is nothing
+else than the ID of one of the predefined filters. This means that in order to
+use the filter inside the table you type its name inside the filter field.
+
+Filter IDs follow these general conventions:
+
+- IDs of filters for a specific record type contain usually the name of a
+  specific group. For instance the ``project::weapons`` filter contains the
+  term ``weapons``. Plural form is always used.
+
+- When filtering a specific subgroup the ID starts just like in the case of
+  general filter. For instance ``project::weaponssilver`` will filter only
+  silver weapons  and ``project::weaponsmagical`` will filter only magical
+  weapons.
+
+- There are few exceptions from the above rule. For instance there are
+  ``project::added``, ``project::removed``, ``project::modified`` and
+  ``project::base``. You might except something more like
+  ``project::statusadded`` but in this case typing these extra characters would
+  not improve readability.
+
+We strongly recommend you take a look at the filters table right now to see
+what you can filter with the defaults. Try using the default filters first
+before writing you own.
+
+
+
+Writing your own filters
+************************
+
+If you want to create your own filter you have to know exactly what do you want
+to get in order to translate this into the expressions. Finally, you will have
+to write this with the correct syntax. As a result table will show only the
+desired rows. This section covers everything you need to know in order to
+create any filter you may want to.
+
+
+Namespaces
+==========
+
+Every default filter has the prefix ``project``. This is a *namespace*, a term
+borrowed from the C++ programming language along with the syntax. In the case
+of the OpenMW CS a namespace always means the scope of said object [#]_. The
+meaning of namespaces is as follows:
+
+``project::``
+   Indicates that the filter is to be used throughout the project in multiple
+   sessions. You can restart the CS and the filter will still be there.
+
+``session::``
+   Indicates that the filter is not stored between multiple sessions and once
+   you quit the OpenMW CS application the filter will be gone. Until then it
+   can be found inside the filters table.
+
+In addition the these two scopes there is a third one called *one-shot*.
```

Comment:
Also, technically one-shot isn't a scope, because one-shot filters aren't actually records. They are just a piece of text.